lib/core/lib/ops/userOps.js

```diff
@@ -513,6 +557,43 @@ function buildUserOps (db/*, config */) {
       })
     },
 
+    /**
+     * List the teams to which the user belongs to
+     * Does not go recursively to parent teams
+     *
+     * @param  {Object}   params { id, organizationId, limit, page }
+     * @param  {Function} cb
+     */
+    listUserTeams: function listUserTeams ({ id, organizationId, page = 1, limit }, cb) {
+      Joi.validate({ id, organizationId, page, limit }, validationRules.listUserTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        const pageLimit = limit || _.get(config, 'authorization.defaultPageSize')
+        const offset = (page - 1) * pageLimit
+
+        const job = {
+          id,
+          organizationId,
+          offset,
+          limit: pageLimit,
+          user: {},
+          client: db
+        }
+
+        readUserTeams(job, (err) => {
+          if (err) return cb(err)
+          const pageSize = pageLimit || job.totalTeamsCount
+          const result = {
+            page,
+            limit: pageSize,
```

Comment:
Why is this returned?

Reply:
I am ok with this return. What problem do you see here?